src/plugins/intel_cpu/src/nodes/psroi_pooling.cpp

```diff
@@ -66,25 +66,20 @@ PSROIPooling::PSROIPooling(const std::shared_ptr<ov::Node>& op, const GraphConte
         OPENVINO_THROW_NOT_IMPLEMENTED(errorMessage);
     }
 
-    errorPrefix = std::string(op->get_type_name()) + " node with name '" + op->get_friendly_name() + "'";
-
     const auto psroi = std::dynamic_pointer_cast<const ov::opset1::PSROIPooling>(op);
     const auto defPsroi = std::dynamic_pointer_cast<const ov::opset1::DeformablePSROIPooling>(op);
 
     noTrans = op->get_input_size() == 2;
     if (op->get_input_shape(0).size() != 4)
-        OPENVINO_THROW(errorPrefix,
-                       " has first input with incorrect rank: " + std::to_string(op->get_input_shape(0).size()));
+        THROW_CPU_NODE_ERR(" has first input with incorrect rank: " + std::to_string(op->get_input_shape(0).size()));
```

Comment:
```suggestion
        THROW_CPU_NODE_ERR("has first input with incorrect rank: " + std::to_string(op->get_input_shape(0).size()));
```
Redundant space, here and below.

Reply:
Removed in all related locations